### PR TITLE
Added coverage test 0442

### PR DESCRIPTION
**Pull request by Vihanakangas**

### CODE REVIEW

File: coverage/PENDING_REVIEW/intel/0442-COVERAGE-nir-opt-comparison-pre-273/cov-and-even-numbers-from-fragcoord.amber [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+#     // Always false.
+#     if(gl_FragCoord.y < _float_0_4)
+#     {
+#         _GLF_color = vec4(_int_0);
+#     }
+#     else
+#     {
+#         // gl_FragCoord has the decimal part of 0.5. The value of v has always
+#         // an even x component and odd y component.
+#         ivec2 v = ivec2((gl_FragCoord.xy - vec2(_float_1_0, _float_0_4)) * _float_256_0);
```
Comment (afd) [inline coverage/PENDING_REVIEW/intel/0442-COVERAGE-nir-opt-comparison-pre-273/cov-and-even-numbers-from-fragcoord.amber:60]: I found the comment hard to follow. Can you spell out more why v will get an even x component and an odd y component?

File: coverage/PENDING_REVIEW/intel/0442-COVERAGE-nir-opt-comparison-pre-273/cov-and-even-numbers-from-fragcoord.amber [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+#         // Anding even numbers always result in zero.
```
Comment (afd) [inline coverage/PENDING_REVIEW/intel/0442-COVERAGE-nir-opt-comparison-pre-273/cov-and-even-numbers-from-fragcoord.amber:61]: Should this say that anding even numbers *by one* always results in zero?

**Review by afd — CHANGES_REQUESTED** [review]

File: coverage/PENDING_REVIEW/intel/0442-COVERAGE-nir-opt-comparison-pre-273/cov-and-even-numbers-from-fragcoord.amber (hunk 1 above)
Comment (Vihanakangas) [reply coverage/PENDING_REVIEW/intel/0442-COVERAGE-nir-opt-comparison-pre-273/cov-and-even-numbers-from-fragcoord.amber]: Updated.

File: coverage/PENDING_REVIEW/intel/0442-COVERAGE-nir-opt-comparison-pre-273/cov-and-even-numbers-from-fragcoord.amber (hunk 2 above)
Comment (Vihanakangas) [reply coverage/PENDING_REVIEW/intel/0442-COVERAGE-nir-opt-comparison-pre-273/cov-and-even-numbers-from-fragcoord.amber]: Updated.

**Comment by Vihanakangas** [comment]
@afd Updated.

**Review by afd — APPROVED** [review]
